src/software/new_geom/util/calculate_block_cone.h

```diff
@@ -0,0 +1,44 @@
+#pragma once
+
+#include "software/new_geom/point.h"
+#include "software/new_geom/vector.h"
+
+/**
+ * Given a cone shooting from the origin, determines the furthest location from
+ * the origin, at which to place a circle to block the cone.
```

Comment:
What does "block the cone" mean here?

Reply:
added comment, it's the cone formed by the vectors a and b presumably